### PR TITLE
Add London Metropolitan University scraper

### PR DESCRIPTION
**Pull request by jodizzle**
Pretty straightforward.

### CODE REVIEW

File: osp_scraper/spiders/londonmet.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            yield scrapy.FormRequest.from_response(
+                response,
+                method="POST",
+                formdata={
+                    'teachinglocation': location_value,
+                    'moduleperiod': "ALL",
+                    'moduletime': "ALL",
+                    'modulelevel': "ALL",
+                    'modulestatus': "ALL",
+                    'academicyear': year,
```
Comment (HarrisonGregg) [inline osp_scraper/spiders/londonmet.py]: Minor: trailing comma

File: osp_scraper/spiders/londonmet.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                },
+                meta={
+                    'depth': 1,
+                    'hops_from_seed': 1,
+                    'source_url': response.url,
+                    'source_anchor': location_anchor + " " + year,
```
Comment (HarrisonGregg) [inline osp_scraper/spiders/londonmet.py]: Minor: trailing comma

File: osp_scraper/spiders/londonmet.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+from ..spiders.CustomSpider import CustomSpider
+
+class LondonmetSpider(CustomSpider):
+    name = "londonmet"
+
+    start_urls = ["https://intranet.londonmet.ac.uk/module-catalogue/"]
+
+    def parse(self, response):
+        # NOTE: The 0th option has value "ALL" and text "North and City", and so
```
Comment (HarrisonGregg) [inline osp_scraper/spiders/londonmet.py]: It does seem like "North and City" is the results from "North campus" and "City campus", but I don't see any harm in leaving them all in there.

Comment (jodizzle) [inline osp_scraper/spiders/londonmet.py]: Yep, that's fine.  I think it's safer to leave it in.

**Pull request merged**